### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by mootpt**
fixing a couple of broken hyperlinks

### CODE REVIEW

File: src/pages/develop/services.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ A railway service is a deployment target for your application. There are two typ
 
 Deployment services can be connected to a GitHub repo and autodeployed on each commit. Templates are a pre-configured groups of services that can be used as a starting off point for your project, or added to an existing project).
 
-Services keep track of all current and historical deployment attempts as well as [variables](/develop/deployments), [metrics](/diagnose/metrics), and application source like a GitHub repo and a start command. Clicking a service within the project canvas will bring up the service view on the side of your browser.
+Services keep track of all current and historical deployment attempts as well as [variables](variables), [metrics](/diagnose/metrics), and application source like a GitHub repo and a start command. Clicking a service within the project canvas will bring up the service view on the side of your browser.
```
Comment (FarazPatankar) [inline src/pages/develop/services.md:17]: Shouldn't this be `/develop/variables`?

File: src/pages/develop/services.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Railway offers 60+ templates that are maintained by the community. A template wi
 
 ## Empty Application Services
 
-Empty services are services with no reference to a repo. They are perfect for [CLI](develop/cli) deploys, running ad-hoc processes, or storing environment variables for local development. A repo can be connected to an empty service at any time on the service settings page.
+Empty services are services with no reference to a repo. They are perfect for [CLI](cli) deploys, running ad-hoc processes, or storing environment variables for local development. A repo can be connected to an empty service at any time on the service settings page.
```
Comment (FarazPatankar) [inline src/pages/develop/services.md:70]: Same thing here. `/develop/cli`?

**Review by FarazPatankar — CHANGES_REQUESTED** [review]

File: src/pages/develop/services.md (hunk 1 above)
Comment (mootpt) [reply src/pages/develop/services.md:17]: hey @FarazPatankar not sure we need the full path in this case, but happy to add

**Review by FarazPatankar — COMMENTED** [review]
Looks good, thanks for the PR! 😄

**Pull request merged**